deployed-devices/rest/certificates/create-certificate/output/create-certificate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "sid": "CYXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "friendly_name": null,
+  "fleet_sid": "FLXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "account_sid": "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "device_sid": "THXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+  "thumbprint": "00112233445566778899aabbccddeeff00112233445566778899aabbccddeeff",
```

Comment:
This string is SHA256 over the supplied certificate data, so it will depend on user input. In theory, they could know upfront what it is, but not mandatory. Happy to rename to <supplied_certificate_fingerprint>